model-events/src/main/java/de/focus_shift/urlaubsverwaltung/extension/api/application/ApplicationDeletedEventDTO.java

```diff
@@ -0,0 +1,39 @@
+package de.focus_shift.urlaubsverwaltung.extension.api.application;
+
+import java.time.Instant;
+import java.time.LocalDate;
+import java.util.Set;
+import java.util.UUID;
+import lombok.Builder;
+import lombok.NonNull;
+import lombok.Value;
+import lombok.extern.jackson.Jacksonized;
+
+@Value
+@Builder
+@Jacksonized
+public class ApplicationDeletedEventDTO {
+  @NonNull private UUID id;
+
+  @NonNull private Instant createdAt;
+
+  @NonNull private String tenantId;
+
+  @NonNull private ApplicationPersonDTO person;
+
+  @NonNull private ApplicationPersonDTO appliedBy;
+
+  private boolean twoStageApproval;
+
+  @NonNull private ApplicationPeriodDTO period;
+
+  @NonNull private VacationTypeDTO vacationType;
+
+  private String reason;
+
+  @NonNull private String status;
+
+  private boolean teamInformed;
```

Comment:
This is a field of the application not of the deletion event itself. Maybe it would be good to model the context of deletion in own applicationDto?

Reply:
why is this different to other application events in case of deletion?